### PR TITLE
Add metabase version override and update

### PR DESCRIPTION
**Pull request by LaurenceJJones**

### CODE REVIEW

**Comment by LaurenceJJones** [comment]
Link to #2369 but adds cscli flag to override version so users are not hardcoded to a single version

**Comment by mmetc** [comment]
Two things - 

1 - the name of the variable would be metabaseVersion, not metabaseDefaultVersion

2 - we should let the user override metabaseImage string instead

**Comment by LaurenceJJones** [comment]
> Two things - 
> 
> 1 - the name of the variable would be metabaseVersion, not metabaseDefaultVersion

Agree

> 
> 2 - we should let the user override metabaseImage string instead

I disagree that the user should be able to pass the whole image name as if an admin allows the user to run as sudo they will be able to run any container

**Comment by mmetc** [comment]
> > 2 - we should let the user override metabaseImage string instead
> 
> I disagree that the user should be able to pass the whole image name as if an admin allows the user to run just cscli as sudo they will be able to run any container

which can be done already with cscli explain

File: cmd/crowdsec-cli/dashboard.go [hunk 1]
```diff
@@ -199,6 +199,7 @@ cscli dashboard setup -l 0.0.0.0 -p 443 --password <password>
 	cmdDashSetup.Flags().BoolVarP(&force, "force", "f", false, "Force setup : override existing files")
 	cmdDashSetup.Flags().StringVarP(&metabaseDbPath, "dir", "d", "", "Shared directory with metabase container")
 	cmdDashSetup.Flags().StringVarP(&metabaseListenAddress, "listen", "l", metabaseListenAddress, "Listen address of container")
+	cmdDashSetup.Flags().StringVarP(&metabaseDefaultVersion, "version", "v", metabaseDefaultVersion, "Metabase version to use")
```
Comment (blotus) [inline cmd/crowdsec-cli/dashboard.go:202]: `metabaseDefaultVersion` should be `metabaseVersion`

**Review by blotus — CHANGES_REQUESTED** [review]
Looks mostly ok to me, but as @mmetc said, the user should be able to override the whole image. (I don't really consider this to be a security risk if they have sudo for cscli, as they can already execute arbitraty commands as root easily).

**Review by blotus — APPROVED** [review]

**Pull request merged**